doc/man/8/ceph-fuse.rst

```diff
@@ -50,6 +50,12 @@ Any options not recognized by ceph-fuse will be passed on to libfuse.
 
    Connect to specified monitor (instead of looking through ceph.conf).
 
+.. option:: -n client.{cephx-username}
+
+   Pass the name of CephX user who's secret key has be to used for mounting.
+   This option is mandatory to when admin keyring is absent on the host where
```

Comment:
The last sentence doesn't seem quite right. You need this when you want your sessions to connect as a different cephx user than "admin" (which is the default).

Reply:
You are right. Fixed it.